[INFO -- 2017-02-26 16:13:47]
Starting flip_value script
version: 2.0
[INFO -- 2017-02-26 16:13:47]
Initialization strings:
[INFO -- 2017-02-26 16:13:47]
(1) target extended-remote | ssh -T mic1 gdbserver --multi -
[INFO -- 2017-02-26 16:13:47]
(2) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-02-26 16:13:47]
(3) set sysroot /opt/mpss/3.7/sysroots/k1om-mpss-linux/
[INFO -- 2017-02-26 16:13:47]
(4) file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-02-26 16:13:47]
(5) set remote exec-file /micNfs/gdb-fault-injection/injector/bin/lud/lud_check
[INFO -- 2017-02-26 16:13:47]
(6) set args -s 2048 -n 228 -i /micNfs/gdb-fault-injection/injector/bin/lud/input_2048_th_228 -g /micNfs/gdb-fault-injection/injector/bin/lud/gold_2048_th_228
[INFO -- 2017-02-26 16:13:47]
initSignal:4
[INFO -- 2017-02-26 16:13:47]
endSignal:8
[INFO -- 2017-02-26 16:13:47]
seqSignal:30
[INFO -- 2017-02-26 16:13:47]
faultMode:2
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[EXCEPTION -- 2017-02-26 16:14:01]
pythonException: invalid frame
[INFO -- 2017-02-26 16:14:01]
Backtrace BEGIN:
#0  0x0000000000404e2d in L_lud_omp_123__par_loop2_2_5 () at lud_omp.c:154
#1  0x00007ffff790ba1f in ?? ()
#2  0x00007fffffffd9c0 in ?? ()
#3  0x00007ffff4ea0100 in ?? ()
#4  0x0000000000000001 in ?? ()
#5  0x0000000000000003 in ?? ()
#6  0x0000000000000000 in ?? ()

149	                    }
150	                }
151	#pragma omp simd 
152	                for (j = 0; j < BS; j++) {
153	                    BB((i+i_global),(j+j_global)) -= sum[j];
154	                    sum[j] = 0.f;
155	                }
156	            }
157	        }
158	    }

Backtrace END
Memory content before bitflip:0xe80x0e0x000x00
Fault Mode: Random bit-flip
base address to flip value: 0x7fffffffc614
address max offset: 4
Memory content after  bitflip:0x740x200xb40x1f
frame name: L_lud_omp_123__par_loop2_2_5
symbol name: chunk_idx
symbol filename: lud_omp.c
symbol line: 45
value: 531898484
value address: 0x7fffffffc614
Type: The type is an integer type.
Type sizeof: 4

[INFO -- 2017-02-26 16:14:01]
Fault Injection Successful after 14.9652240276s
[INFO -- 2017-02-26 16:14:11]
event type: exit
[INFO -- 2017-02-26 16:14:11]
exit code: 1
